File: MicroservicesProject/src/SubscriptionService/springboot-app/src/main/java/com/SubscriptionMicroservice/springbootapp/Responses/BaseResponse.java
package com.SubscriptionMicroservice.springbootapp.Responses;

public class BaseResponse {

    public String Sender;
    public String ResponseType;
    public String Message;

    public BaseResponse(String sender, String responseType, String message) {
        Sender = sender;
        ResponseType = responseType;
        Message = message;
    }
}

File: MicroservicesProject/src/WPF_Client/ClientBackLogic/Responses/BaseResponse.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ClientBackLogic.Responses
{
    public class BaseResponse
    {
        public string Sender { get; set; }
        public string ResponseType { get; set; }
        public string Message { get; set; }
    }
}
